src/libraries/Common/src/Interop/Interop.Odbc.cs

```diff
@@ -17,14 +17,14 @@ internal static partial class Odbc
         //
         // ODBC32
         //
-        [DllImport(Interop.Libraries.Odbc32)]
-        internal static extern /*SQLRETURN*/ODBC32.RetCode SQLAllocHandle(
+        [GeneratedDllImport(Interop.Libraries.Odbc32)]
+        internal static partial ODBC32.RetCode SQLAllocHandle(
```

Comment:
I wonder if we should rename RetCode to SQLRETURN?

Reply:
Renamed.